client/src/components/landing/AboutMistakesJournal.vue

```diff
@@ -76,9 +96,15 @@
           <span class="step-text">{{ $t('LandingPage.AboutMistakesJournal.Step3.Text2') }}</span>
         </div>
         <img
+          v-if="$i18n.locale === 'EN'"
+          alt="Mistakes journal front page"
+          class="mj-landing-steps-image"
+          src="@/assets/icons/mistakes_en.svg">
```

Comment:
With all those images I would do something like

```
        <img
          v-if="$i18n.locale === 'EN'"
          alt="Mistakes journal front page"
          class="mj-landing-steps-image"
          :src="require(@/assets/icons/' + $i18n.locale + '/mistakes.svg)">
```
and place the images in respective dirctories: `icons/PL` and `icons/EN`, without language name in the file.